wam/settings.py

```diff
@@ -205,6 +205,6 @@
         pass
 
 # E-mail config for outgoing mails (used by exchangelib)
-WAM_EXCHANGE_ACCOUNT = config['WAM']['WAM_EXCHANGE_ACCOUNT']
-WAM_EXCHANGE_EMAIL = config['WAM']['WAM_EXCHANGE_EMAIL']
-WAM_EXCHANGE_PW = config['WAM']['WAM_EXCHANGE_PW']
+WAM_EXCHANGE_ACCOUNT = config['WAM'].get('WAM_EXCHANGE_ACCOUNT')
+WAM_EXCHANGE_EMAIL = config['WAM'].get('WAM_EXCHANGE_EMAIL')
+WAM_EXCHANGE_PW = config['WAM'].get('WAM_EXCHANGE_PW')
```

Comment:
Git workflow comment : 
This fix will be committed in this PR. However it is not related to the PR's purpose, nor to the issue the PR addresses. This could(should?) theorically trigger an issue creation and a separate PR to fix the issue. That way the changes made to the repository are easier to read and understand for humans :). 

I think the rule can be bent for small changes like that one. Especially if they are already committed, because it would cost time to undo the changes here and create a new issue/branch/PR.

Reply:
Yeah you are right - I had this in my repo for longer and did not want to start PR first.
Anyway, what is supposed way to enter such small changes? PR?

Reply:
> Yeah you are right - I had this in my repo for longer and did not want to start PR first.
> Anyway, what is supposed way to enter such small changes? PR?

To my mind, you could commit small changes directly to dev. In general: if in doubt, go for a PR which should be approved quickly